### DISCUSSION TITLE
Where is ` and ~ with the QWERTY alternative layout (ZMK)

### DISCUSSION THREAD

**Discussion by psilospore** [manna-harbour/miryoku#302] [Q&A] [answered]
Hi I've been enjoying this layout and apologies if this is obvious but I haven't been able to find the ` and ~ symbol with my layout.

I also wasn't able to find the exact layout. The one without qwerty should have it on the z key so when I do sym I should get tilde but instead I have ± and when I do Num I should get ` but instead it's §. I wonder if that's a mistake since those symbols seem kin of rare?

This is my build option (nice_nano_v2, corne_right/left, QWERTY, vi, Win, default, default, default, default, default)] running on a chocifi

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/discussions/197#discussioncomment-5240943.
